pkg/controller/cloud/node_controller.go

```diff
@@ -168,6 +169,11 @@ func (cnc *CloudNodeController) updateNodeAddress(node *v1.Node, instances cloud
 		glog.Errorf("%v", err)
 		return
 	}
+
+	if len(nodeAddresses) == 0 {
```

Comment:
This allows for providers to do updates to node addresses if they want.

Reply:
can we please log that we are skipping updates using `glog.V(5).Infof`?